### PR TITLE
Feature/actions

### PR DESCRIPTION
**Pull request by szczygiel-m**
Added actions for:

- Delete topic
- Delete subscription
- DC Readiness switch
- Delete topic in consistency view
- Delete group
- Suspend/activate subscription

### CODE REVIEW

File: hermes-console-vue/src/views/subscription/subscription-metadata/SubscriptionMetadata.spec.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { beforeEach } from 'vitest';
+import { dummyOwner } from '@/dummy/topic';
 import { dummySubscription } from '@/dummy/subscription';
 import { expect } from 'vitest';
 import { render } from '@/utils/test-utils';
```
Comment (moscicky) [inline hermes-console-vue/src/views/subscription/subscription-metadata/SubscriptionMetadata.spec.ts:5]: minor: we can consider adding tests for checking whether events were emitted on btn clicks

File: hermes-console-vue/src/composables/topic/use-topic/useTopic.spec.ts [hunk 2]
```diff
@@ -173,6 +190,48 @@ describe('useTopic', () => {
...
+    // then
+    await waitFor(() => {
+      expectNotificationDispatched(notificationStore, {
+        type: 'success',
+        text: 'notifications.topic.delete.success',
+      });
+    });
+  });
+
+  it('should show message that removing subscription was unsuccessful', async () => {
```
Comment (moscicky) [inline hermes-console-vue/src/composables/topic/use-topic/useTopic.spec.ts:214]:
```suggestion
  it('should show message that removing topic was unsuccessful', async () => {
```

File: hermes-console-vue/src/components/confirmation-dialog/ConfirmationDialog.spec.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      props,
+    });
+
+    // then
+    expect(
+      getByText('confirmationDialog.confirm').closest('button'),
+    ).toBeDisabled();
+  });
+
+  it('should require confirmation text', () => {
```
Comment (moscicky) [inline hermes-console-vue/src/components/confirmation-dialog/ConfirmationDialog.spec.ts:62]: nice

File: hermes-console-vue/src/composables/dialog/use-dialog/useDialog.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+export function useDialog(
+  isOpenedByDefault: boolean = false,
+  actionButtonEnabledByDefault: boolean = true,
+): UseDialog {
+  const isDialogOpened = ref(isOpenedByDefault);
+  const actionButtonEnabled = ref(actionButtonEnabledByDefault);
+
+  async function openDialog() {
```
[the remark this answers is not in the document]
Comment (szczygiel-m) [reply hermes-console-vue/src/composables/dialog/use-dialog/useDialog.ts:20]: fixed

File: hermes-console-vue/src/views/admin/consistency/inconsistent-topics-listing/InconsistentTopicsListing.vue [hunk 5]
```diff
@@ -13,6 +13,8 @@
       (topic) => !props.filter || topic.includes(props.filter),
     );
   });
+
+  defineEmits(['remove']);
```
[the remark this answers is not in the document]
Comment (szczygiel-m) [reply hermes-console-vue/src/views/admin/consistency/inconsistent-topics-listing/InconsistentTopicsListing.vue:17]: done

File: hermes-console-vue/src/views/admin/consistency/ConsistencyView.vue [hunk 6]
```diff
@@ -29,9 +35,49 @@
       title: t('consistency.breadcrumbs.title'),
     },
   ];
+
+  const topicToDelete = ref();
+
+  const {
+    isDialogOpened: isRemoveDialogOpened,
+    actionButtonEnabled: actionRemoveButtonEnabled,
```
[the remark this answers is not in the document]
Comment (szczygiel-m) [reply hermes-console-vue/src/views/admin/consistency/ConsistencyView.vue:43]: fixed

**Review by moscicky — APPROVED** [review]

**Pull request merged**